package.json

```diff
@@ -77,6 +77,7 @@
     "prop-types": "^15.6.0",
     "react": "^16.2.0",
     "react-bootstrap": "^0.31.5",
+    "react-debounce-input": "^3.2.0",
```

Comment:
@ekohl ack/nack?

Reply:
Because we branched very late this was already included so :+1: from a packaging perspective